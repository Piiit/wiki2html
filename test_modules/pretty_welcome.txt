{{authorA:="Lorenzo"}}
{{authorB:="Peter"}}
= This is an example generated with wiki2html=
Welcome from {{authorA}} & {{authorB}}!
==What is that?==
This is a **wiki** parser basically, where you can easily produce HTML-compliant files in a go.
==Is that free?==
Yes it is absolutely //free// and //open source//.
==Who wrote it?==
{{authorA}} & {{authorB}} for a Compilers course project.