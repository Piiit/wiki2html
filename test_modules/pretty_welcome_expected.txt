<!doctype html>
<html>
<head>
<title>This is an example generated with wiki2html</title>
</head>
<body>
<div id='toc'><b>Table of content:</b><br />
<ul>
<li><a href='#header_0'>This is an example generated with wiki2html</a></li>
<li><a href='#header_1'>What is that?</a></li>
<li><a href='#header_2'>Is that free?</a></li>
<li><a href='#header_3'>Who wrote it?</a></li>
</ul></div>
<h1 id='header_0'>This is an example generated with wiki2html</h1>
Welcome from Lorenzo & Peter!
<h2 id='header_1'>What is that?</h2>
This is a <b>wiki</b> parser basically, where you can easily produce HTML-compliant files in a go.
<h2 id='header_2'>Is that free?</h2>
Yes it is absolutely <i>free</i> and <i>open source</i>.
<h2 id='header_3'>Who wrote it?</h2>
Lorenzo & Peter for a Compilers course project.
</body>
</html>

